packages/dcos-launch/extra/launch/ip-detect/aws_public.sh

```diff
@@ -1,4 +1,11 @@
 #!/bin/sh
 set -o nounset -o errexit
 
+if [ -e /etc/environment ]
+then
+  set -o allexport
```

Comment:
I had to look around to know the meaning of this `bash` set options. It will be great it a comment was provided for the reader. Enable exporting from file; export properties from file, and then disable exporting from file).